test/__snapshots__/scenarios.test.js.snap

```diff
@@ -239,7 +239,122 @@ body {
+exports[`Layout and 3 podlets - run 6 - minification and all entrypoints run 1`] = `
+"!(function t(n, r, u) {
+  function e(o, c) {
+    if (!r[o]) {
+      if (!n[o]) {
+        var f = \\"function\\" == typeof require && require;
+        if (!c && f) return f(o, !0);
+        if (i) return i(o, !0);
+        var s = new Error(\\"Cannot find module '\\" + o + \\"'\\");
+        throw ((s.code = \\"MODULE_NOT_FOUND\\"), s);
+      }
+      var p = (r[o] = { exports: {} });
+      n[o][0].call(
+        p.exports,
+        function(t) {
+          var r = n[o][1][t];
+          return e(r || t);
+        },
+        p,
+        p.exports,
+        t,
+        n,
+        r,
+        u
+      );
+    }
+    return r[o].exports;
+  }
+  for (
+    var i = \\"function\\" == typeof require && require, o = 0;
+    o < u.length;
+    o++
+  )
+    e(u[o]);
+  return e;
+})(
+  {
+    1: [
+      function(t, n, r) {
+        \\"use strict\\";
+        t(\\"./d\\")(5, 6);
+      },
+      { \\"./d\\": 2 }
+    ],
+    2: [
+      function(t, n, r) {
+        \\"use strict\\";
+        n.exports = function(t, n) {
+          return t + n;
+        };
+      },
+      {}
+    ],
+    3: [
+      function(t, n, r) {
+        \\"use strict\\";
+        t(\\"./c\\");
+        spy(\\"h\\");
+      },
+      { \\"./c\\": 1 }
+    ],
+    4: [
+      function(t, n, r) {
+        arguments[4][2][0].apply(r, arguments);
+      },
+      { dup: 2 }
+    ],
+    5: [
+      function(t, n, r) {
+        \\"use strict\\";
+        t(\\"./d\\")(10, 6);
+      },
+      { \\"./d\\": 4 }
+    ],
+    6: [
+      function(t, n, r) {
+        \\"use strict\\";
+        t(\\"./d\\")(1, 2);
+      },
+      { \\"./d\\": 8 }
+    ],
+    7: [
+      function(t, n, r) {
+        \\"use strict\\";
+        t(\\"./d\\")(3, 4);
+      },
+      { \\"./d\\": 8 }
+    ],
+    8: [
+      function(t, n, r) {
+        arguments[4][2][0].apply(r, arguments);
+      },
+      { dup: 2 }
+    ],
+    9: [
+      function(t, n, r) {
+        \\"use strict\\";
+        t(\\"./a\\");
+        spy(\\"f\\");
+      },
+      { \\"./a\\": 6 }
+    ],
+    10: [
+      function(t, n, r) {
+        \\"use strict\\";
+        t(\\"./b\\");
+        spy(\\"g\\");
+      },
+      { \\"./b\\": 7 }
+    ]
+  },
+  {},
+  [5, 9, 10, 3]
```

Comment:
5 is layout server, 9 and 10 are multi-entry points for 1 podlet, 3 is another podlet with single entrypoint.

If you look through you can see that 9 and 10 share the same version of 2 where as 5 and 3 get their own versions though in none of the cases is 2 actually duplicated. `arguments[4][2][0].apply(r, arguments);` is used to make groups that all point back at 2.

Reply:
awesome